yapf/yapflib/format_decision_state.py

```diff
@@ -336,8 +336,8 @@ def SurroundedByParens(token):
     ###########################################################################
     # Argument List Splitting
     if (style.Get('SPLIT_BEFORE_NAMED_ASSIGNS') and not current.is_comment and
-        format_token.Subtype.DEFAULT_OR_NAMED_ASSIGN_ARG_LIST in
-        current.subtypes):
+        format_token.Subtype.DEFAULT_OR_NAMED_ASSIGN_ARG_LIST in current
+        .subtypes):
```

Comment:
This change has me a bit worried. While we can split before the `.` it's probably preferable to split before `current`. Are you able to distinguish between this case and the one in your description?

Reply:
looks like these changes were introduced in
https://github.com/google/yapf/commit/49003a872a8fa3383791ec1cb46392d429636c7b

things got confused beacuse of a rebase..